.eslintrc.json

```diff
@@ -6,19 +6,28 @@
     "es6": true,
     "mocha": true
   },
+    "parser": "babel-eslint",
     "parserOptions": {
-      "ecmaVersion": 2020,
+      "ecmaVersion": 2015,
```

Comment:
Why ecma 2015 ?

Reply:
> Why ecma 2015 ?

I was getting an error message about the ECMA version because env is using es6. So I changed this to 2015 (same as es6)